Client/Frontend/Browser/Search/SearchEngines.swift

```diff
@@ -261,7 +261,8 @@ class SearchEngines {
     /// Get all known search engines, possibly as ordered by the user.
     fileprivate func getOrderedEngines() -> [OpenSearchEngine] {
         let locale = Locale(identifier: Locale.preferredLanguages.first ?? Locale.current.identifier)
-        let unorderedEngines = customEngines + SearchEngines.getUnorderedBundledEnginesFor(locale: locale)
+        let prefs = [Preferences.Search.defaultEngineName, Preferences.Search.defaultPrivateEngineName].compactMap { $0.value }
```

Comment:
This name is not clear imo, I had to take a moment to understand what it is.
Maybe rename it, and method parameters you added to something like `selectedSearchEngines`?